gnomad_qc/v5/data_ingestion/federated_validity_checks.py

```diff
@@ -84,20 +84,17 @@ def validate_ht_fields(ht: hl.Table, config: Dict[str, Any]) -> None:
     :param config: JSON configuration for parameter inputs.
     :return: None.
     """
-    indexed_array_annotations = {
-        config["freq_fields"]["freq"]: config["freq_fields"]["freq_index_dict"],
-    }
+    # Define array struct annotations.
+    array_struct_annotations = [config["freq_fields"]["freq"]]
 
     if config.get("faf_fields"):
-        indexed_array_annotations[config["faf_fields"]["faf"]] = config["faf_fields"][
-            "faf_index_dict"
-        ]
+        array_struct_annotations.append(config["faf_fields"]["faf"])
```

Comment:
should there be a note here that the expected name for the faf field in the config is "faf"?